### PR TITLE
Added eslint and prettier check on stories

### PR DESCRIPTION
**Pull request by mlipardo**

### CODE REVIEW

**Comment by mlipardo** [comment]
#30 added a dependency "react-autosuggest" and yarn lock was not committed.

**Comment by eckdanny-osi** [comment]
@mlipardo @edgibbs I suggest we keep topic branches atomic. I'm happy to raise an issue and submit a PR to resolve the issue discovered with #30. If we glob issues in PRs, it will be difficult to introspect (e.g.; to answer questions like: _Have we experienced enough missing lockfile PRs to warrant a git precommit hook?_)

Maintainers plz weigh in. 

Meanwhile I'll create an issue and submit PR to resolve.

**Comment by eckdanny-osi** [comment]
PR to fix lockfile (#39) is awaiting review. Plz remove `yarn.lock` changes from this PR.

File: package.json [hunk 1]
```diff
@@ -18,10 +18,10 @@
   ],
   "scripts": {
     "prepublish": "babel --plugins transform-es2015-modules-umd src --out-dir ./dist && node-sass src/styles/application.scss dist/styles/application.scss && node-sass src/styles/application.scss dist/styles/application.css && echo 'all dist copied.'",
-    "lint": "eslint ./src",
+    "lint": "eslint ./src && eslint ./stories",
```
Comment (eckdanny-osi) [inline package.json]: consider using a glob instead of unix AND operator

**Pull request merged**